plover/machine/txbolt.py

```diff
@@ -11,7 +11,7 @@
 # byte represents. The next bits are set if the corresponding key was
 # pressed for the stroke.
 
-# 00XXXXXX 01XXXXXX 10XXXXXX 110XXXXX
+# 00XXXXXX 01XXXXXX 10XXXXXX 11XXXXXX
 #   HWPKTS   UE*OAR   GLBPRF    #ZDST
```

Comment:
I don't think this should be changed, as the line was correct: the last set has only 5 keys.